_posts/2022-02-28-war-in-ukraine.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "The War in Ukraine"
+author: [michal.kosmulski]
+tags: [tech, "soft skills", helping]
+---
+The attack on Ukraine has shocked the whole world, but in Poland it is perceived in an especially poignant way.
+Not only is Ukraine our direct neighbor (as are Russia and Belarus), but Poland’s difficult history allows us to
+sympathize with the Ukrainians even more. Many live in Poland, so it’s affecting people we know personally.
+
+While the fighting continues, here’s what [Allegro](https://allegro.tech) is doing, and what you can do, too, to help
+humanitarian efforts.
+
+* If you know any Ukrainians, your coworkers in particular, talk to them and show your support. Even minor gestures are
+  important. Ask directly what kind of help they might need rather than trying to guess.
+* Do not marginalize Russians and Belorussians. People are not the same as their governments, and often do not support
+  their states’ actions.
+* Work with your HR people and consider what kind of help you can offer as a company. Some examples would be extra days
+  off, legal and financial support for bringing families to safety, or offering psychological support.
+* On a personal level, consider supporting one of the trustworthy humanitarian organizations. You can support them
+  with cash or by donating necessary items which are being collected in most cities.
+* Ask if the company can [donate directly, make donations by others easier](https://spolecznosc.allegro.pl/t5/blog/wsparcie-dla-ukrainy-co-mo%C5%BCesz-zrobi%C4%87-na-allegro-aby-pom%C3%B3c/ba-p/334998) [PL],
+  provide means of communication, etc., depending on the industry.
```

Comment:
Zamieniłabym te dwa punkty miejscami, tj. żeby "Work with your HR people" i "Ask if the company can donate directly" były zgrupowane razem.